### PR TITLE
add kakoune editor

### PR DESCRIPTION
**Pull request by freevryheid**
thanks for maintaining this!

### CODE REVIEW

File: docs/editor_integration.rst [hunk 1]
```diff
@@ -96,3 +96,29 @@ Installing this `VS17 extension`_ should enable ``fortls`` features in Visual St
...
+
+Install `kak-lsp <https://github.com/kak-lsp/>`_.
+
+Edit the kak-lsp.toml config file to include:
+
+.. code-block:: sh
+
+  [language.fortran]
+  filetypes = ["fortran"]
+  roots = [".git", ".fortl
```
Comment (gnikit) [inline docs/editor_integration.rst:110]: This looks like it should be ".fortls"]

Comment (freevryheid) [reply docs/editor_integration.rst:110]: that's correct.

**Comment by gnikit** [comment]
Nice job @freevryheid and thanks for the contribution. Is this PR done?

**Comment by freevryheid** [comment]
Yes, that's it for now.

**Comment by gnikit** [comment]
Cool I will merge then. Thanks again!

**Pull request merged**